dissect/util/stream.py

```diff
@@ -199,9 +209,12 @@ def __init__(self, fh: BinaryIO, offset: int, size: int, align: int = STREAM_BUF
         self.offset = offset
 
     def _read(self, offset: int, length: int) -> bytes:
+        self._seek_locked = True
```

Comment:
I'm not sure it's necessary to set these in the subclasses, since all calls to `_read` are issued from `AlignedStream.read` where you already set the `self._seek_locked`.